code/datums/mood_events/generic_negative_events.dm

```diff
@@ -431,8 +431,6 @@
 	description = "I've been here too long! I want to go out and explore space!"
 	mood_change = -3
 
-///Wizard cheesy grand finale - what everyone but the wizard gets
-/datum/mood_event/madness_despair
-	description = "UNWORTHY, UNWORTHY, UNWORTHY!!!"
-	mood_change = -200
-	special_screen_obj = "mood_despair"
+/datum/mood_event/rust_corruption
+	description = "Oh god everything is falling apart, the station isnt safe..."
+	mood_change = -5
```

Comment:
You need to restore this.

Reply:
what does that mean

Reply:
it means you deleted a different mood event for no reason and now your code doesn't compile